### PR TITLE
Add Russell.java, implements Student interface

### PR DESCRIPTION
**Pull request by russell-stewart**

### CODE REVIEW

**Comment by MrClement** [comment]
Great work. Maybe it would be good to have some instance data, but his is still fine.

**Comment by akelani16** [comment]
hello!

**Comment by slevy16** [comment]
👍

**Comment by DrewParsons** [comment]
ayyyyyyyyy

**Comment by akelani16** [comment]
fahim is life.

File: Russell.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  }
+
+  public String getFavoriteFood()
+  {
+    return "Barbecue and Turnip Greens";
+  }
+
+  public void sayCatchphrase()
+  {
+    System.out.println("nyoooooooom");
```
Comment (MrClement) [inline Russell.java:15]: This catchphrase should be capitalized!

**Pull request merged**